libs/core/src/components/pds-avatar/test/pds-avatar.e2e.ts

```diff
@@ -0,0 +1,38 @@
+import { newE2EPage } from '@stencil/core/testing';
+
+describe('pds-avatar', () => {
+  it('renders', async () => {
+    const page = await newE2EPage();
+    await page.setContent('<pds-avatar></pds-avatar>');
+
+    const element = await page.find('pds-avatar');
+    expect(element).toHaveClass('hydrated');
+  });
+
+  it('renders admin variant with correct class', async () => {
+    const page = await newE2EPage();
+
+    await page.setContent('<pds-avatar variant="admin"></pds-avatar>');
```

Comment:
nitpick ⚠️ :  move the line up and have newline after this. Similar to what you have on lines 5-8. Apply to the rest of the file, please and thanks.